fsys_test.go

```diff
@@ -0,0 +1,135 @@
+package skeletonkit_test
+
+import (
+	"embed"
+	"flag"
+	"io"
+	"path/filepath"
+	"strings"
+	"testing"
+	"text/template"
+
+	"github.com/gostaticanalysis/skeletonkit"
+	"github.com/tenntenn/golden"
+)
+
+type appInfo struct {
+	Name       string
+	ModulePath string
+}
+
+type errMap struct {
+	errParse  bool
```

Comment:
I think prefix `err` does not need because these field are referred with `tt.wantErr`.